Formula/mpd.rb

```diff
@@ -84,12 +87,12 @@ def install
       -Dvorbisenc=enabled
     ]
 
-    system "meson", *args, "output/release", "."
-    system "ninja", "-C", "output/release"
+    system "meson", "setup", "output/release", *args, *std_meson_args
+    system "meson", "compile", "-C", "output/release"
     ENV.deparallelize # Directories are created in parallel, so let's not do that
```

Comment:
I don't think this does anything, since it only changes `MAKEFLAGS`, and we use `ninja` now. Unless `ninja` also checks `MAKEFLAGS`?